.github/workflows/bump.yml

```diff
@@ -48,21 +47,37 @@ jobs:
           passphrase: ${{ secrets.PHYLUM_BOT_GPG_PASSPHRASE }}
           git_user_signingkey: true
           git_commit_gpgsign: true
-          git_config_global: true
 
-      - name: Set up Homebrew
-        uses: Homebrew/actions/setup-homebrew@master
+      - name: Update URL and SHA256 values
+        id: commit
+        continue-on-error: true
+        # NOTE: The `archive/refs/tags` link is used instead of the `tarball_url` provided by the release API because
+        #       the hashes between the two are (currently) different, even though the extracted contents are the same.
+        #       The `archive/refs/tags` link format is used by the formula and is therefore the one downloaded here.
+        #       Reference: https://github.com/community/community/discussions/45830
+        # NOTE: The git user name/email used for commits is already configured, by crazy-max/ghaction-import-gpg action.
+        run: |
+          curl -sSfLo "cli-${CLI_VER#v}.tar.gz" "https://github.com/phylum-dev/cli/archive/refs/tags/${CLI_VER}.tar.gz"
```

Comment:
We could just use `-O`/`--remote-name` here with `curl` to use `${CLI_VER}.targ.gz` as name automatically.